src/components/MaterialUI/DataDisplay/TransferList.js

```diff
@@ -0,0 +1,197 @@
+import List from "@material-ui/core/List";
+import ListItem from "@material-ui/core/ListItem";
+import ListItemText from "@material-ui/core/ListItemText";
+import Button from "@material-ui/core/Button";
+import Paper from "@material-ui/core/Paper";
+import Typography from "@material-ui/core/Typography";
+import classNames from "classnames";
+import { isReactComponent } from "../../../utils/propertyValidator";
+import sharedMessages from "../../../sharedMessages";
+
+const useStyles = makeStyles(theme => ({
+	root: {
+		margin: "auto",
+	},
+
+	title: {
+		color: theme.palette.text.hint,
+		fontSize: theme.spacing(1.2),
+		marginBottom: theme.spacing(0.6),
+	},
+	paper: {
+		overflow: "auto",
+		width: theme.spacing(32.6),
+		height: theme.spacing(48),
+		border: `${theme.spacing(0.1)} solid ${theme.palette.primary.light}`,
+		borderRadius: theme.spacing(0.5),
+		boxShadow: "none",
+
+		"&::-webkit-scrollbar": {
+			width: theme.spacing(1.5),
+		},
+		"&::-webkit-scrollbar-thumb": {
+			background: theme.palette.grey.borders,
+			border: `${theme.spacing(0.5)} white solid`,
+			backgroundClip: "padding-box",
+			borderRadius: theme.spacing(1.5),
+		},
+	},
+	paperLeft: {
+		border: `${theme.spacing(0.1)} solid ${theme.palette.grey.borders}`,
+	},
+	item: {
+		fontSize: theme.spacing(1.3),
+		color: theme.palette.text.primary,
+		fontFamily: theme.typography.fontFamily,
+		padding: theme.spacing(0, 1.6),
+		cursor: "pointer",
+	},
+	selectedItem: {
+		backgroundColor: theme.palette.primary.light,
+	},
+
+	button: {
+		margin: theme.spacing(0.5, 0),
+		fontSize: theme.spacing(1.3),
+		lineHeight: theme.spacing(1.4),
+		padding: theme.spacing(0.7),
+		minWidth: theme.spacing(8),
+	},
+}));
+
+const formatState = (state, checked, from, to) => {
+	const fromSelectedItems = state[from].filter(({ id }) => checked.includes(id));
+	return {
+		[to]: [...state[to], ...fromSelectedItems].sort((a, b) => a.title.localeCompare(b.title)),
+		[from]: state[from].filter(({ id }) => !checked.includes(id)),
+	};
+};
+
+const CustomList = ({ items, checked, setChecked, listItemFormatter }) => {
+	const classes = useStyles();
+
+	const handleToggle = useCallback(
+		value =>
+			setChecked(state => {
+				const currentIndex = state.indexOf(value);
+				const newChecked = [...state];
+
+				if (currentIndex === -1) {
+					newChecked.push(value);
+				} else {
+					newChecked.splice(currentIndex, 1);
+				}
+				return newChecked;
+			}),
+		[setChecked],
+	);
+
+	return (
+		<List component="div" role="list">
+			{items.map(item => (
+				<ListItem
+					className={classNames(classes.item, checked.includes(item.id) && classes.selectedItem)}
+					key={item.id}
+					role="listitem"
+					onClick={!item.disabled ? () => handleToggle(item.id) : null}
+					disabled={item.disabled}
+				>
+					{listItemFormatter ? (
+						listItemFormatter(item)
+					) : (
+						<ListItemText
+							key={`transfer-list-item-${item.id}-label`}
+							primary={item.title}
```

Comment:
is it possible to make by default the stylw for primary title like:
fontWeight: theme.typography.fontWeightSemiBold